packages/consumption/test/modules/requests/DecideRequestParametersValidator.test.ts

```diff
@@ -154,11 +131,11 @@ describe("DecideRequestParametersValidator", function () {
             },
             expectedError: {
                 code: "error.consumption.requests.decide.validation.invalidNumberOfItems",
-                message: "Number of items in Request and Response do not match"
+                message: "Number of items in Request and Response do not match."
             }
         },
         {
-            description: "(2) error: Request with one item is answered with two items",
+            description: "(2) error: Request with one RequestItem is answered with two items",
```

Comment:
Request with one RequestItem is answered with two **ResponseItems**

Reply:
I struggle with this formulation because, strictly speaking, you do not respond with a ResponseItem, but with certain DecideRequestItemParameters, from which ResponseItems are then generated if the Request is successfully answered. Nevertheless, I do not like this wording either, so if you have an alternative suggestion that avoids the problem I mentioned, please let me know.